cogs/vxtwitconverter/eventHandlers.py

```diff
@@ -0,0 +1,10 @@
+from discord import Message
+
+from cogs.vxtwitconverter.eventsCore import EventCore
```

Comment:
Cog doesn't load, need the following changes:
```suggestion
from .eventsCore import EventCore
```